applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_two_fluid_solver.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.mpi as KratosMPI                          # MPI-python interface
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("FluidDynamicsApplication","MetisApplication","TrilinosApplication")
+
+# Import applications
+import KratosMultiphysics.MetisApplication as KratosMetis           # Partitioning
+import KratosMultiphysics.TrilinosApplication as KratosTrilinos     # MPI solvers
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid   # Fluid dynamics application
+
+# Import serial two fluid solver
+import navier_stokes_two_fluids_solver
+import trilinos_import_model_part_utility
+
+def CreateSolver(model, custom_settings):
+    return NavierStokesMPITwoFluidSolver(model, custom_settings)
+
+class NavierStokesMPITwoFluidSolver(navier_stokes_two_fluids_solver.NavierStokesTwoFluidsSolver):
+
+    def _ValidateSettings(self, settings):
+
+        default_settings = KratosMultiphysics.Parameters("""{
+            "solver_type": "TwoFluids",
+            "model_part_name": "",
+            "domain_size": -1,
+            "model_import_settings": {
+                "input_type": "mdpa",
+                "input_filename": "unknown_name"
+            },
+            "distance_reading_settings"    : {
+                "import_mode"         : "from_GID_file",
+                "distance_file_name"  : "distance_file"
+            },
+            "maximum_iterations": 7,
+            "dynamic_tau": 1.0,
+            "echo_level": 0,
+            "consider_periodic_conditions": false,
+            "time_order": 2,
+            "compute_reactions": false,
+            "reform_dofs_at_each_step": false,
+            "relative_velocity_tolerance": 1e-3,
+            "absolute_velocity_tolerance": 1e-5,
+            "relative_pressure_tolerance": 1e-3,
+            "absolute_pressure_tolerance": 1e-5,
+            "linear_solver_settings"       : {
+                "solver_type"                        : "MultiLevelSolver",
+                "max_iteration"                      : 200,
+                "tolerance"                          : 1e-6,
+                "max_levels"                         : 3,
+                "symmetric"                          : false,
+                "reform_preconditioner_at_each_step" : true,
+                "scaling"                            : true
+            },
+            "volume_model_part_name" : "volume_model_part",
+            "skin_parts": [""],
+            "no_skin_parts":[""],
+            "time_stepping": {
+                "automatic_time_step" : true,
+                "CFL_number"          : 1,
+                "minimum_delta_time"  : 1e-4,
+                "maximum_delta_time"  : 0.01
+            },
+            "periodic": "periodic",
```

Comment:
After investigating closer, it seems that this entry is no longer longer needed or even provokes "depreciated" messages in some cases. Thus, it is removed. In contrast, the entry "consider_periodic_conditions" remains since it is vital for the domain decomposition.